### ISSUE TITLE
[Issue]: Continue Watching Doesn't Remember the Version Played

### ISSUE THREAD

**Issue by buzzzz2138** [jellyfin/jellyfin#11118]
### Please describe your bug

I have a folder that has the following files inside of it:
'Dune.1984 - 1Original.en.srt'                     
'Dune.1984 - 1Original.mp4'     
'Dune.1984 - 2Extended.en.srt'
'Dune.1984 - 2Extended.mp4'   
'Dune.1984 - 3Bonus Disc.mp4'
'Dune.1984 - 4Alternative Edition Redux.mp4'
'Dune.1984.trailer.mp4'

Under Version I see the following.
![image](https://github.com/jellyfin/jellyfin/assets/3680372/9eb6dbfb-f329-458d-afee-9b39f84251d6)

### Reproduction Steps

If I got into the movie and select the 2Extended Version it plays as expected.  If I then stop watching and later continue watching from the home page, it starts the 4Alternative Edition Redux version. 
The work around is to go into the movie page and start the continue watching with the correct version selected.  Continue Watching doesn't remember the version that was played. 

### Jellyfin Version

10.8.13

### Environment

```markdown
- OS: Debian 11 (Open Media Vault)
- Linux Kernel:
- Virtualization: Docker
- Clients: Firefox 123
- Browser: Firefox 123
- FFmpeg Version: Docker
- Playback Method: Continue Watching
```

### Jellyfin logs

```shell
NA
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jbatt33** [comment]
Checking the data repository, we only have playbackPositionTicks storeted. We are storing the AudioStreamIndex and the SubtitleStreamIndex, but I'm not sure if these are updated when the item was started. Checking more.

**Comment by SourSulfur** [comment]
I'm not sure if there's a specific reason why the local alternate versions of a movie are not assigned the parent ID but assigning it does make them resumable.

If this is indeed a satisfactory change, I'm unsure if the place at which I handled this is ok. There might be a better place to handle this at and could use suggestions/feedback on this.

**Comment by SourSulfur** [comment]
While replicating this I've also noticed that going into the movie page and selecting the version only works if that's the only version that has been played.

So, if I play Version 1 and then resume V1 from the movie page, it resumes from the right spot. However, if I now play V2 till some other point midway and then try to resume V1 again from the movie page, it plays from V2's stop point, not V1.

I was wondering if someone else could confirm if they can replicate this behavior or if there's already an issue that tracks this?

**Comment by buzzzz2138** [comment]
The stop point is recorded but isn't related to the version of movie being played only the Movie page. 
Play V1 to 20 minutes.
Then stop and go back to the movie page and select/play V2 and it'll start at the 20 minute mark.  That reads to me that the current position time is only related to the movie as a whole and not related to the version.

**Comment by SourSulfur** [comment]
Yeah, I'm assuming this is a bug and the current position should be associated to each version. Not sure if there's already an issue for this if it is indeed a bug.

**Comment by Shadowghost** [comment]
If the userdata supports differentiating between versions, it's definitly a bug. If not, it's tech debt left over from bad Emby implementations.

**Comment by SourSulfur** [comment]
It does, seems like it needs a change on jellyfin web. I'll take a look.

**Comment by MaximeLussier** [comment]
I have the same issue but cannot seem to continue the version I started, even from the movie page. The "Continue Watching" section on the home page has the movie, but with no progress bar...

Going to the movie page and selecting the right version also starts from the beginning.
